packages/frontend/app/services/graphql.js

```diff
@@ -1,5 +1,4 @@
 import Service from '@ember/service';
-import fetch from 'fetch';
```

Comment:
I feel like this was important, but can't remember exactly why. Making a note for discussion.

Reply:
it's the only place that i found where were using the polyfill instead of the browser-native implementation of `fetch()`.
if need be, we could add `ember-fetch` as dependency to frontend.